addons/resource/models/resource_resource.py

```diff
@@ -62,7 +62,7 @@ def _compute_avatar_128(self):
     @api.model_create_multi
     def create(self, vals_list):
         for values in vals_list:
-            if values.get('company_id') and not values.get('calendar_id'):
+            if values.get('company_id') and not 'calendar_id' in values:
```

Comment:
I'm confused (perhaps I'm going crazy) but how is that changing anything ? ``values.get(x)`` should return ``None`` if x is not in values and ``None`` evaluates to ``False``. We get exactly the same result when doing ``'calendar_id' in values`` right ? or did I miss something ? :thinking:

Reply:
Hello @arg-odoo thank you very much for the reply. Honestly, perfect explanation and quite the oversight on my end :muscle: :partying_face: :top: